dev-python/typogrify/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
Adding a package as maintainer-needed is not allowed.

@gentoo/gnome and/or @gentoo/python might maintain if it the test suite was wired up.